test/meta_test.js

```diff
@@ -169,4 +169,26 @@ suite('Repo Meta Tests', function () {
       throw new Error(errors);
     }
   });
+  test("no references to tools.taskcluster.net in the repository", async function () {
```

Comment:
@subhamskm Looks like this change is irrelevant to this PR and is instead related to #961

Can you please remove it?

Reply:
Actually, you can drop the commits https://github.com/taskcluster/taskcluster/pull/974/commits/2c957e3475a695685ac7357aa607a04d5ef7f54c and https://github.com/taskcluster/taskcluster/pull/974/commits/0bc61e950f244bd550f9c080d8f0fb376c4d7160. Let me know if you need any help with it.